hlxsites/merative2
# feat(tag-picker): Enable search + copy tags interface for Content Authors

## Issue

Fix #191 

## Test URLs
  
- Before: https://main--merative2--hlxsites.hlx.page/
- After: https://191-tagpicker--merative2--hlxsites.hlx.page/tools/tagger/index.html
  
## How to test

Besides being it's own page (instructions are on it), a link was added to the sidekick for the tagger.
To test, temporarily edit the sidekick's Github URL to end in /tree/191-tagpicker. 
Navigate to a Sharepoint page, and you should see a new button called "tagger" which opens the tagging tool in a new browser tab.
![image](https://github.com/hlxsites/merative2/assets/3883795/13640904-56d2-4292-bc0f-e0f1538a1472)
